student.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, name, _parental_permission, classroom)
+    super(age, name)
```

Comment:
- [ ] [OPTIONAL] Kindly pass the `_parental_permission` to the superclass so that it will be like:
```
super(age, name, parental_permission: _parental_permission)
```